src/exporters/era5_land.py

```diff
@@ -0,0 +1,178 @@
+from pathlib import Path
+from typing import Optional, Dict, List, cast
+import warnings
+import multiprocessing
+import itertools
+
+from .cds import CDSExporter
+from .base import get_kenya
+
+VALID_ERA5_LAND_VARS = [
+    '10m_u_component_of_wind', '10m_v_component_of_wind',
+    '2m_dewpoint_temperature', '2m_temperature',
+    'evaporation_from_bare_soil',
+    'evaporation_from_open_water_surfaces_excluding_oceans',
+    'evaporation_from_the_top_of_canopy',
+    'evaporation_from_vegetation_transpiration',
+    'evapotranspiration', 'forecast_albedo', 'lake_bottom_temperature',
+    'lake_ice_depth', 'lake_ice_temperature', 'lake_mix_layer_depth',
+    'lake_mix_layer_temperature', 'lake_shape_factor',
+    'lake_total_layer_temperature', 'leaf_area_index_high_vegetation',
+    'leaf_area_index_low_vegetation', 'potential_evaporation', 'runoff',
+    'skin_reservoir_content', 'skin_temperature', 'snow_albedo', 'snow_cover',
+    'snow_density', 'snow_depth', 'snow_depth_water_equivalent',
+    'snow_evaporation', 'snowfall', 'snowmelt', 'soil_temperature_level_1',
+    'soil_temperature_level_2', 'soil_temperature_level_3',
+    'soil_temperature_level_4', 'sub_surface_runoff',
+    'surface_latent_heat_flux', 'surface_net_solar_radiation',
+    'surface_net_thermal_radiation', 'surface_pressure', 'surface_runoff',
+    'surface_sensible_heat_flux', 'surface_solar_radiation_downwards',
+    'surface_thermal_radiation_downwards', 'temperature_of_snow_layer',
+    'total_precipitation', 'volumetric_soil_water_layer_1',
+    'volumetric_soil_water_layer_2', 'volumetric_soil_water_layer_3',
+    'volumetric_soil_water_layer_4'
+]
+
+
+class ERA5LandExporter(CDSExporter):
+    dataset = 'reanalysis-era5-land'
+    granularity = 'hourly'
+
+    @staticmethod
+    def print_valid_vars():
```

Comment:
Is this function necessary? It's not used

Reply:
i used it alot to work out what requests to send to the `export()` function

my workflow was like:

```python
>>> exporter.print_valid_vars()
>>> exporter.export(<copied_valid_variable>)
```

because the names have to be exactly right for the function to work